robot_skills/src/robot_skills/arm/grasp_sensor.py

```diff
@@ -0,0 +1,127 @@
+from robot_skills.robot_part import RobotPart
+from tue_manipulation_msgs.msg import GripperCommandGoal, GripperCommandAction
+from tue_msgs.msg import GripperCommand
+
+class GripperMeasurement(object):
+    """
+    Class holding measurements from the distance sensor on the grippers
+    """
+    EMPTY = -1
+    UNKNOWN = 0
+    HOLDING = 1
+
+    def __init__(self, distance):
+        """
+        Constructor
+
+        :param distance: float with measured distance
```

Comment:
This doesn't match our current style of docstrings in which we use `@param xxx` etc.

Reply:
This was a part of code that was moved. It was already there